### ISSUE TITLE
can only select one file at a time in file dialog

### ISSUE THREAD

**Issue by jeisner** [Fauntleroy/neutral-face-emoji-tools#23]
Hard to upload 1000+ emoji when I have to click on them individually.

I'm using Chrome on Linux (with a MATE desktop).  Sometimes file dialogs allow multiple selection using Shift-click and Ctrl-click, but this one doesn't.  Am I doing something wrong?  Would I be able to select multiple files if I used another browser or OS?

**Comment by particledecay** [comment]
I'm having the same issue, Fedora 36 on Gnome (Xorg). I can only select one file at a time.

**Comment by shuuji3** [comment]
You're not wrong. This is because `<input type="file">` tag has no `multiple` attribute. We can default select only one file in the file selector dialog. The `multiple` allows us to select multiple files. I'll send a PR to fix this issue.

**Comment by spencer-shadley** [comment]
This issue still exists. I fixed it here though: https://github.com/Fauntleroy/neutral-face-emoji-tools/pull/40 please accept the PR
